arviz/tests/external_tests/test_data_pyro.py

```diff
@@ -70,6 +78,11 @@ def test_inference_data(self, data, eight_schools_params, predictions_data):
         fails = check_multiple_attrs(test_dict, inference_data)
         assert not fails
 
+        # test dims
+        dims = inference_data.posterior_predictive.dims["school"]
+        pred_dims = inference_data.predictions.dims["school_pred"]
+        assert dims == 8, pred_dims == 8
```

Comment:
these should still be one in each line, otherwise the `pred_dims == 8` will only be executed if assert dims fails.